tv.plex.PlexHTPC.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Plex HTPC
+Comment=Plex HTPC for Linux is your client for playing on your computer connected to the big screen
+Exec=/usr/bin/flatpak run --branch=master --arch=x86_64 tv.plex.Htpc
```

Comment:
definitely NOT. This line should be the plain executable name that can be found in `/app/bin`

Reply:
Fixed